skyportal/tests/api/test_shifts.py

```diff
@@ -23,6 +30,10 @@ def test_shift(public_group, super_admin_token, view_only_token, super_admin_use
     assert status == 200
     assert data['status'] == 'success'
 
+    assert any([request_data['name'] == s['name'] for s in data["data"]])
     assert any([request_data['start_date'] == s['start_date'] for s in data["data"]])
-
     assert any([request_data['end_date'] == s['end_date'] for s in data["data"]])
+    assert (
+        len([s for s in data['data'][0]['users'] if s['id'] == super_admin_user.id])
```

Comment:
Can't guarantee that your shift will be at index 0. would probably need to do two loops, one with `if s['name']==name` and one inside that to check the users.

Reply:
Oh yes you're very right i tried that first and forgot to add the 2nd loop. Will do it now